css-showcases/src/transition-helper.js

```diff
@@ -43,7 +43,7 @@ export const getTransitionsHtml = (element, props, mode) => {
     .map(
       ([name, value], i) => /*html*/ `
       <div id="transitionBox${i}"
-          class="${styles.box} ${
+          class="not-prose ${styles.box} ${
```

Comment:
Where is not-prose coming from?
If it comes from our layouts it shouldn't be used in `dockit-core`

Reply:
it's here to fix style collisions with `prose` from our docs layout, but encapsulation with shadow dom is the better fix as mentioned in other comments, I'll go for that

Reply:
Fixed this differently, by dropping `pre` and just styling, encapsulation with shadow dom for this component and it's parent is a bigger beast that we might address in a PR of its own.